README.md

```diff
@@ -941,4 +942,26 @@ function custom_menu_order( $menu_ord ) {
 }
 add_filter( 'custom_menu_order', 'custom_menu_order' );
 add_filter( 'menu_order', 'custom_menu_order' );
+```
+
+## Exclude a category from WordPress loops
+
+```php
+/**
+ * Exclude a category from all WordPress loops
+ */ 
+add_action( 'pre_get_posts', function ($query ) { // anonymous callback
+	
+	global $wp_query; 
+
+	// Hard coded category ID, but can be dynamic: esc_attr(get_option('your-cat-id')); 
+	$excluded_cat_id = 25;
+
+	// add category ID to existing, avoid overwritting it 
+	$cat[] = $query->get( 'cat' );
+	$cat[] = "-" . $webtv_category_parent;
```

Comment:
What is `$webtv_category_parent` here? I believe that's a leftover from the original code in a particular project.

Reply:
Yeah @ihorvorotnov, definitely, I fixed the typo. Thanks for pointing that out.